haechan45th/src/controllers/userController.js

```diff
@@ -10,6 +10,30 @@ const getAllUsers = async (req, res) => {
     }
 }
 
+const signIn = async (req, res) => {
+    try {
+        const { email, password } = req.body;
+
+        if (!email || !password) {
+            return res.status(400).json({ message: 'KEY_ERROR' });
+        }
+
+        const { token, user } = await userService.signIn(email, password);
+        res.cookie('user', token, { httpOnly: true, sameSite: 'none', secure: true });
+
+        return res.status(200).json({
+            message: 'SIGNIN_SUCCESS',
+            token,
+            user
+        });
```

Comment:
```suggestion
        const accessToken = await userService.signIn(email, password);

        return res.status(200).json({
            message: 'SIGNIN_SUCCESS',
            accessToken
        });
```
1. cookie는 사용하지 않으므로 해당 코드는 지워주세요!
2. user data는 로그인시 보내주지 않습니다. 오로지 accessToken을 주고 받으며 해당 토큰은 서버에서 사용자별 고유값으로 식별자로서 넣어놓은 데이터가 있기에 이를 통해 토큰의 주인을 식별할 수 있습니다.